app/_layouts/default.html

```diff
@@ -29,17 +29,17 @@
 
     {% include_cached footer.html %} {% include_cached anchor_links.html %}
 
-    {% if jekyll.environment == "production" %}
+    <script src="/assets/app.js?v={{ site.time | date: '%s' }}"></script>
+    {% if jekyll.environment != "production" %}
```

Comment:
shouldn't this be
```suggestion
    {% if jekyll.environment == "production" %}
```

Reply:
This is why we have a review process! That was me testing locally and I forgot to revert the change